### PR TITLE
managed-gitops: Update gitops component to use new kustomize-based manifest organization 

### PR DESCRIPTION
**Pull request by jgwest**
Update gitops component to use new kustomize-based manifest organization

### CODE REVIEW

**Comment by jgwest** [comment]
https://github.com/redhat-appstudio/managed-gitops/pull/303

File: components/gitops/staging/kustomization.yaml [hunk 1]
```diff
@@ -2,80 +2,13 @@ apiVersion: kustomize.config.k8s.io/v1beta1
...
-- https://raw.githubusercontent.com/redhat-appstudio/managed-gitops/aa8214ccb5e9c1c530848bd8aafdba98cb98e5e2/manifests/postgresql-staging/postgresql-staging.yaml
-
-# Route(s)
-- https://raw.githubusercontent.com/redhat-appstudio/managed-gitops/aa8214ccb5e9c1c530848bd8aafdba98cb98e5e2/manifests/routes/routes.yaml
-
-# 'gitops-service-argocd' Namespace
-- https://raw.githubusercontent.com/redhat-appstudio/managed-gitops/aa8214ccb5e9c1c530848bd8aafdba98cb98e5e2/manifests/staging-cluster-resources/argo-cd-namespace.yaml
 
 images:
   - name: \${COMMON_IMAGE}
```
[the remark this answers is not in the document]
Comment (Michkov) [reply components/gitops/staging/kustomization.yaml:9]: https://github.com/redhat-appstudio/managed-gitops/blob/main/backend/config/manager/manager.yaml#L45

**Comment by jgwest** [comment]
Appears to be a failure on openshift-ci setup.

/retest

**Comment by jgwest** [comment]
Fail in pipeline test

/retest

**Comment by jgwest** [comment]
Cluster claim fail

/retest

**Comment by jgwest** [comment]
/retest

Cluster pool issue

**Comment by jgwest** [comment]
/devfile_private_source.go:146
  [FAIL] [e2e-demos-suite] RHDP-467: create an application with a component from private and public quay.io images [It] create component quay-public-component from public container source [e2e-demo]
  /tmp/tmp.8SKSgNrBWz/tests/e2e-demos/e2e-demo.go:157
  [FAIL] [e2e-demos-suite] multi-component-application [It] Check multiple components are deployed [e2e-demo]
  /tmp/tmp.8SKSgNrBWz/tests/e2e-demos/multi-component.go:221

Ran 128 of 149 Specs in 1343.587 seconds
FAIL! -- 124 Passed | 4 Failed | 7 Pending | 14 Skipped

**Pull request merged**